src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,42 @@
+
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+
+    private static final String White_Space = " ";
+    private static final int Date_Index = 0;
+    private static final int Name_Index = 1;
+    private static final int Hours_Index = 2;
+    private static final int Salary_Index = 3;
```

Comment:
Constant names use CONSTANT_CASE: all uppercase letters, with each word separated from the next by a single underscore